kiwix.yml

```diff
@@ -511,7 +511,7 @@ all:
     url: "http://download.kiwix.org/portable/gutenberg/kiwix-0.9+gutenberg_es_all_10_2014.zip"
     name: Project Gutenberg
     language: "spa"
```

Comment:
Because of different origins AFAIK.
This key is unused at the moment, and we should switch to `languages` soon, I suggest we'll clean this by then.